### PR TITLE
Review process for new contributions

### LINKED ISSUE #186: Contribution guidelines: review
Write the review process in the contribution guidelines.

### PR DESCRIPTION
**Pull request by ccarouge**
# CABLE

Thank you for submitting a pull request to the CABLE Project.

## Description

Guidelines for the review of new contributions.

Fixes #186 

## Type of change

Please delete options that are not relevant.

- [X] New or updated documentation

## Checklist

- [x] The new content is accessible and located in the appropriate section.
- [x] I have checked that links are valid and point to the intended content.
- [x] I have checked my code/text and corrected any misspellings

Please add a reviewer when ready for review.

----
📚 Documentation preview 📚: https://cable--289.org.readthedocs.build/en/289/

### CODE REVIEW

**Comment by ccarouge** [comment]
@bschroeter When writing this, I realised having a quick explanation of the PR interface in the CABLE documentation would be good. But I think it needs to come earlier in the contribution guidelines so I'm not including it with this work. 

The pages modified by this PR can be found [here](https://cable--289.org.readthedocs.build/en/289/developer_guide/contribution/review_guide/) and [here](https://cable--289.org.readthedocs.build/en/289/developer_guide/contribution/resources/how_to/) in the rendered documentation.

File: documentation/docs/developer_guide/contribution/review_guide.md [hunk 1]
```diff
@@ -1,5 +1,59 @@
...
+- the description of the pull request is up-to-date.
+- all the required test results are either linked to in comments or copied in. If you have performed tests beyond the required tests, make sure these tests and their results are described in comments in your pull request.
+- the automated checks pass. Contact the @CABLE-LSM/admin team, through a comment in your pull request, if you need help understanding a failure of these tests.
+- all conflicts have been solved.
+
+## Asking for review
+
+Once you are ready, [ask for a review by `CABLE-LSM/reviewers`][how-ask-review]. If you want a specific individual, you can use the same process to choose that person. Using the reviewers team is preferred as it will spread the workload across reviewers.
+
+In most cases, asking for one reviewer is enough. The system allows asking for multiple reviewers, please use this feature will parsimony. It is often easier to ask for a single reviewer first and add another reviewer later as necessary.
```
Comment (bschroeter) [inline documentation/docs/developer_guide/contribution/review_guide.md:27]:
```suggestion
In most cases, asking for one reviewer is enough. The system allows asking for multiple reviewers, please use this feature sparingly. It is often easier to ask for a single reviewer first and add another reviewer later as necessary.
```

Small typo, simplified language.

**Review by bschroeter — CHANGES_REQUESTED** [review]
Looks good - Just that one line by the looks of it.

**Review by bschroeter — APPROVED** [review]

**Comment by bschroeter** [comment]
Looks good, might want to check that markdown link check thing.

**Comment by ccarouge** [comment]
The markdown checker is failing on files that have been removed from main but still exist in the outdated documentation in the PR branch. These will be removed from the documentation once merged with main.

**Pull request merged**